update-price-table/update-price-table.py

```diff
@@ -32,30 +37,99 @@ def get_database_connection() -> connection:
         return error
 
 
-def get_product_data(rds_conn:connection):
-    """Query database for data on all products that price data is required for"""
+def create_ses_client():
```

Comment:
Very minor point - could use a `config` argument here and pass in `environ` when called later